packages/webcert/src/store/question/questionReducer.ts

```diff
@@ -59,6 +61,7 @@ const getInitialState = (): QuestionState => {
     isSendingQuestion: false,
     functionDisablers: [],
     errorId: '',
+    isLoadingQuestions: true,
```

Comment:
It a bit counterintuitive to let initial state to be true, it would be better to ensure correct behavior with initial state set to false

Reply:
Then I have to have undefined in the beginning which I have gotten comments to avoid. Which is to be preferred out of the two?

Reply:
What I'm suggesting is that the model matches the expectation of the data. I'm not sure what the client's code looks like but my thinking is that `isLoadingQuestions` should only be able to be `true` or `false`, and only `true` if questions actually are loading. That might mean that some additional checks need to occur in the client to handle state correctly.

Reply:
Fixed this